core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1800,6 +1800,15 @@ private[spark] object Utils extends Logging {
     }
   }
 
+  lazy val isInInterpreter: Boolean = {
```

Comment:
I think what @preeze means, is that pwendell requested it this way since it will also be needed by https://issues.apache.org/jira/browse/SPARK-7261.

But I agree, `Utils` is getting unwieldy.  I was going to suggest putting it in one of the repl classes, but then I realized we don't want to make core depend on repl.  I dunno if there is a better place at the moment.  perhaps we should just open something separate to reorganize `Utils` in logical components?  A quick scan makes me think that we could at least divide into serialization utils, file utils, network utils, randomization utils, unit conversion utils, exception handling, ...